apps/src/templates/lessonOverview/LessonAgenda.jsx

```diff
@@ -0,0 +1,45 @@
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import i18n from '@cdo/locale';
+import _ from 'lodash';
+
+export default class LessonAgenda extends Component {
+  static propTypes = {
+    activities: PropTypes.array.isRequired
+  };
+
+  render() {
+    // Do not link to sections that are progressions and sections without a displayName
+    let filteredActivitiesList = _.cloneDeep(this.props.activities);
+    filteredActivitiesList.forEach(activity => {
+      let filteredActivitySections = [];
+      activity.activitySections.forEach(section => {
+        if (section.displayName !== '' && section.scriptLevels.length < 1) {
+          filteredActivitySections.push(section);
+        }
+      });
+      activity.activitySections = filteredActivitySections;
```

Comment:
This can be simplified a bit:
```suggestion
      activity.activitySections = activity.activitySections.filter(section => (
        section.displayName !== '' && section.scriptLevels.length < 1
      ));
```